src/liballoc/tests/str.rs

```diff
@@ -483,8 +483,8 @@ mod slice_index {
     }
 
     #[test]
-    #[cfg(not(target_os = "emscripten"))] // hits an OOM
-    #[cfg(not(miri))] // Miri is too slow
+    #[cfg_attr(target_os = "emscripten", ignore)] // hits an OOM
```

Comment:
For consistency, I also adjusted the emscripten exclusion here. This is also how emscripten is [excluded elsewhere](https://github.com/rust-lang/rust/pull/67119/files#diff-edfb0a1aa643ade97922064c64652dc0R31).